src/main/java/com/jorndev/jorndevdemo/dto/UserDTO.java

```diff
@@ -4,12 +4,20 @@
 import com.jorndev.jorndevdemo.models.User;
 import lombok.Getter;
```

Comment:
I would recommend to use Lombok for User and Product entities as well, as it's been used here.